subsys/bluetooth/mesh/sensor_srv.c

```diff
@@ -893,6 +926,10 @@ static int sensor_srv_init(struct bt_mesh_model *model)
 
 	srv->pub.update = update_handler;
 	srv->pub.msg = &srv->pub_buf;
+
+	/** Always use fast period to make the server sample sensors with the fastests cadence. */
```

Comment:
I didn't get this comment.

Reply:
When `fast_period` is set to `true`, the access layer will divide a publication period to a fast cadence period divisor. That divisor will always be set to the highest value. This will make sensor server react on sensor changes as fast as possible. When the divisor is zero, `fast_period` will have no affect on the update callback.